WcaOnRails/app/models/competition.rb

```diff
@@ -424,7 +425,7 @@ def belongs_to_region?(region)
   end
 
   def contains?(search_param)
-    [name, cityName, venue, cellName, countryId, start_date.strftime('%B')].any? do |field|
+    [name, cityName, delegates.map(&:name).join(','), venue, cellName, countryId, start_date.strftime('%B')].any? do |field|
```

Comment:
Changed to pluck.
At first I read "more flexible" different from what you meant. I thought you would search for 2 words and get all records that matched any of those, but what you have is only those that match *both* words.
I guess I'll leave it as it is now and wait for people to start using and detect potential needs.